.gitignore

```diff
@@ -1,12 +1,7 @@
 *.iml
 .gradle
 /local.properties
-/.idea/caches
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
-/.idea/navEditor.xml
-/.idea/assetWizardSettings.xml
+.idea
```

Comment:
I don't think that we should remove all `.idea` folder, the default config should be fine and I think if they did put this folder in VCS it is for a reason, though I don't know which.

Reply:
I put it back again but just removed the .idea/androidTestResultsUserPreferences.xml that is not useful and just creates conflicts during merges for nothing